### ISSUE TITLE
Improve bulk product image upload UX

### ISSUE THREAD

**Issue by myriamboure** [openfoodfoundation/openfoodnetwork#2170]
## Description
While testing https://github.com/openfoodfoundation/openfoodnetwork/pull/1464 new feature to upload image in bulk product edit page, I noticed that the upload UX was a bit user unfriendly.

## Expected Behavior

Still being discussed so wait until we have agreed on a solution before starting!!!

**Option 1:**
- If there is no image I click on image frame, open modal with “choose image”, I choose, it upload in the preview modal and I have two options “change image” or “confirm”. If click “change image” I am asked to choose again an image. If I click on “confirm” the modal closes and the image displays in the product bulk edit list.
- If there is an image I can click on it either to see it bigger or because I want to change it. So if I click on an existing image should have the preview modal open with only one button “change image” (and ability to close the modal of course)

**Option 2:**
- If there is no image I click on image frame, open modal with “choose image”, I choose, it uploads, closes the modal directly and displays instantly in the product bulk edit list without the preview step. (less clicks is better)
- If there is an image I can click on it either to see it bigger or because I want to change it. So if I click on an existing image should have the preview modal open with only one button “change image” (and ability to close the modal of course). That way I can also check how my image looks like if I want, but it makes the process quicker for those who just want to go quick.

## Actual Behavior
When upload a image in bulk product edit page, display preview but modal remains open with "upload an image" button (confusing and one more click needed)

## Steps to Reproduce
1. On "products", bulk edit product page
2. Click on an empty image frame on the first column, upload an image and see UX

## Animated Gif/Screenshot
![screenshot from 2018-03-23 13-01-10](https://user-images.githubusercontent.com/8135376/37828117-4ec59918-2e9a-11e8-8847-924444e55b29.png)

## Context
Noticed that while testing bulk edit product image upload PR1464

## Severity
s3

## Your Environment
- Version used: branch PR1464 deployed on UK stagig
- Browser name and version: Version 64.0.3282.167 (Official Build) (64-bit)
- Operating System and version (desktop or mobile): Ubuntu 16.04 LTS

## Possible Fix
- either no preview, closes the modal as soon as image is chosen
- either preview but the two button choices: "change image" or "accept"
Sill in discussion

**Comment by myriamboure** [comment]
Assigning that to you @Matt-Yorkley but wait a bit until we agree on the best UX solution before starting.

**Comment by RachL** [comment]
@myriamboure I'm in favor of option 2 :) Also, it is important that when you click anywhere on the page the modale closes. But I guess it is already the case.

**Comment by myriamboure** [comment]
@sstead @daniellemoorhead what do you think? Does option 2 sounds ok to you? Should we ask Yuko as well or is that clear enough for everyone? Thanks :-)

**Comment by daniellemoorhead** [comment]
Will leave to @sstead and potentially @oeoeaio to have an opinion :-)

**Comment by myriamboure** [comment]
@sstead @oeoeaio any opinion on that? Rachel and I proposed option 2. Current UX for this new feature doesn't seem satisfying to me.

**Comment by sauloperez** [comment]
I experienced the same here. I thought having a "close" button would help so I bet for option 1 but I'm not an expert.

**Comment by myriamboure** [comment]
I'm just wondering @sauloperez as for me less clicks the better, and you always see the small icon so you know the upload happened, not sure you need the preview here... but let's see what the others say ;-)

**Comment by sauloperez** [comment]
Yes, it was just my feeling at that moment but I prefer some other authoritative voice here. Agree with you that less clicks the better.

**Comment by sstead** [comment]
I support option 2

**Comment by myriamboure** [comment]
@RachL had created a new issue to describe precisely the story after this discussion and avoid any confusion https://github.com/openfoodfoundation/openfoodnetwork/issues/2214, so I'm closing this one and linking the new one to the PR.
